tests/benchmark/benchmark_queries.py

```
@@ -0,0 +1,194 @@
+        times = []
+        for run in range(1, repeats + 1):
+            server = pycer.PyOfflineServer()
+            server.declare_stream(declaration_text)
+            if options_text:
+                server.declare_option(options_text)
+            server.add_query(query_text)
+
+            # --- Timed section: only C++ engine work ---
+            start = time.perf_counter()
+            if has_timing:
+                for event, delay_ns in zip(events, delays_ns):
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)